test/client/fixtures/core/page-unload-barrier-test.js

```diff
@@ -101,4 +102,52 @@ $(document).ready(function () {
                 start();
             });
     });
+
+    module('Regression');
+
+    if (browserUtils.isIE9 || browserUtils.isIE10) {
```

Comment:
I've got error on IE 11 when tested without this condition locally with Sauce Labs - "Access is denied"
https://assets.saucelabs.com/jobs/ee41e00cebdf41c48f42ddb9ebfaff6f/0037screenshot.png